javascript-data-model/order-history.js

```diff
@@ -0,0 +1,69 @@
+var orderHistory = [
+  {
+    placed: 'August 4, 2020',
+    total: '$34.00',
+    to: 'JS Masher',
+    order: '114-3941689-8772232',
+    details: '',
+    invoice: '',
+    delivered: 'Aug 8, 2020',
+    image: '',
+    title: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    return: 'Return window closed on Sep 7, 2020',
+    price: '31.55',
+    again: '',
+    view: '',
+    review: ''
+  },
+  {
+    placed: 'July 19, 2020',
+    total: '$44.53',
+    to: 'JS Masher',
+    order: '113-9984268-1280257',
+    details: '',
+    invoice: '',
+    delivered: 'Jul 20, 2020',
+    status: 'Your package was delivered. It was handed directly to a resident.',
+    image: '',
+    title: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    return: 'Return window closed on Aug 19, 2020',
+    price: '41.33',
+    again: '',
+    view: '',
+    review: ''
+  },
+  {
+    placed: 'July 4, 2020',
+    total: '$17.22',
+    to: 'JS Masher',
+    order: '114-2875557-9059409',
+    details: '',
+    invoice: '',
+    delivered: 'Jul 7, 2020',
+    status: 'Your package was delivered. It was handed directly to a resident',
+    image: '',
+    title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Apater---',
+    return: 'Return window closed on Aug 5, 2020',
+    again: '',
+    view: '',
+    review: ''
+  },
+  {
+    placed: 'July 5, 2020',
+    total: '$138.93',
+    to: 'JS Masher',
+    order: '113-2883177-268248',
+    details: '',
+    invoice: '',
+    delivered: 'Jul 5, 2020',
+    title: ['GameCube Controller', 'The Art of Sql'],
+    return: 'Return window closed on Aug 4, 2020',
+    price: ['$94.95, 33.99'],
```

Comment:
Not ice how the value here is an array, but in your other orders, you have just a string saved. This is not consistent. If you need an array for this order, the rest of your orders should also include arrays for the sake of consistency